docs/faq.md

```diff
@@ -0,0 +1,252 @@
+
+* config.json: configuration that is being evaluated in the trial
+* std.err: standard error 
+* std.out: standard output
+
+In addition all checkpointing files used by a training script such as intermediate model checkpoint will also be located there.
+This is exemplified in the following example:
+
+```
+tree ~/syne-tune/train-height-2022-01-12-11-08-40-971/
+~/syne-tune/train-height-2022-01-12-11-08-40-971/
+├── 0
+│   ├── config.json
+│   ├── std.err
+│   ├── std.out
+│   └── stop
+├── 1
+│   ├── config.json
+│   ├── std.err
+│   ├── std.out
+│   └── stop
+├── 2
+│   ├── config.json
+│   ├── std.err
+│   ├── std.out
+│   └── stop
+├── 3
+│   ├── config.json
+│   ├── std.err
+│   ├── std.out
+│   └── stop
+├── metadata.json
+├── results.csv.zip
+└── tuner.dill
+```
+
+### <a name="reporter-metrics"></a> Where can I find the output of the tuning?
+
+When running locally, the output of the tuning is saved under `~/syne-tune/{tuner-name}/`. 
+When running remotely, the tuning output is synced regularly to  `s3://{sagemaker-default-bucket}/syne-tune/{tuner-name}/`.
+Note: If you run remote tuning via the CLI, the tuning output is synced to `s3://{sagemaker-default-bucket}/syne-tune/{experiment-name}/{tuner-name}/`, where `experiment-name` is the prefix of `tuner-name` without the datetime extension (in the example above, `experiment-name = 'train-height'`).
+
+### <a name="plotting-tuning"></a> How can I plot the results of a tuning?
+
+The easiest way to plot the result of a tuning experiment is to call the following:
+
+```
+tuner = Tuner(
+    ...
+    tuner_name="plot-results-demo",
+)
+tuner.run()
+tuning_experiment = load_experiment(tuner.name)
+tuning_experiment.plot()
+```
+
+This generates a plot of the best value found over time. Note that this you can also plot the results while the experiment is running as results are updated continuously.
+
+### <a name="additional-metadata"></a> How can I specify additional tuning metadata?
+
+By default, Syne Tune stores the time, the names and modes of the metric being tuner, the name of the entrypoint, the name backend and the scheduler name.
+You can also add custom metadata to your tuning job by setting `metadata` in `Tuner` as follow:
+
+```
+tuner = Tuner(
+    ...
+    tuner_name="plot-results-demo",
+    metadata={"tag": "special-tag", "user": "alice"},
+)
+```
+
+All Syne Tune and user metadata are saved when the tuner starts under `metadata.json`.
+
+
+### <a name="logging-additional-information"></a> How do I append additional information to the results which are stored? 
+
+Results are processed and stored by callbacks passed to `Tuner`, in particular see https://github.com/awslabs/syne-tune/blob/main/syne_tune/tuner_callback.py#L80. In order to add more information to these results, you can inherit from `StoreResultsCallback`. A good example is given in https://github.com/awslabs/syne-tune/blob/main/syne_tune/optimizer/schedulers/searchers/searcher_callback.py#L63.
```

Comment:
Would be nice to have shorter links [short](full). Sorry, maybe I did not do this, but did not know how i Quip.